src/plugin/ipc/event/eventconnection.h

```diff
@@ -74,21 +74,18 @@ class EpollConnection : public Connection
   public:
     enum EpollType {
       EPOLL_INVALID = Connection::EPOLL,
-      EPOLL_CREATE,
-      EPOLL_CTL,
-      EPOLL_WAIT
+      EPOLL_REGULAR,
     };
 
-    EpollConnection(int size, int type = EPOLL_CREATE)
+    EpollConnection(int size = 0, int flags = 0)
       : Connection(EPOLL),
-      _type(type),
-      _size(size)
+      _type(EPOLL_REGULAR),
```

Comment:
Are we using the `_type` field in any meaningful way now? If not, should we just remove it?